pages/docs/workflows.md

```diff
@@ -296,33 +290,20 @@ Exactly as per the VSCode Extension above, external module imports via `<script
 
 The full Node.js resolver rules are supported so it's even possible to use [own-name resolution](https://nodejs.org/dist/latest-v17.x/docs/api/packages.html#self-referencing-a-package-using-its-name) and [package imports resolution](https://nodejs.org/dist/latest-v17.x/docs/api/packages.html#subpath-imports) while respecting custom local import map mappings.
 
-## JSPM Starter
-
-While the previous workflows all show isolated examples of import map generation, the [JSPM Starter repo](https://github.com/jspm/jspm-starter) provides a full featured development example with TypeScript support.
-
-It is recommended to use [Chomp Task Runner](https://chompbuild.com) instead of npm scripts for the starter, which provides Makefile-like incremental builds with caching as well as a dev server.
 
-### Chomp Setup
+# Development Workflows
 
-Chomp can be installed either via npm:
-
-```sh
-npm install -g chomp
-```
+Import maps can be used in development workflows for Browser, Node and Deno environments. Browser application workflows are not required to have a bundling step, and all environments may still employ file watching and hot-reloading. Equally, build steps involving transpilation and using task runners are supported when using import maps. Simply incorporate your preferred method for generating import maps with your preferred workflow.
```

Comment:
```suggestion
Import maps can be used in development workflows for Browser, Node.js and Deno environments. Browser application workflows are not required to have a bundling step, and all environments may still employ file watching and hot-reloading. Equally, build steps involving transpilation and using task runners are supported when using import maps. Simply incorporate your preferred method for generating import maps with your preferred workflow.
```

It's a shame we don't have hot-reloading workflows ready to demo... well we do, but they are still an unreleased Chomp feature.